Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -21,12 +24,12 @@
                 sum without (%(etcd_instance_labels)s) (up{%(etcd_selector)s} == bool 0)
               or
                 count without (To) (
-                  sum without (%(etcd_instance_labels)s) (rate(etcd_network_peer_sent_failures_total{%(etcd_selector)s}[1m])) > 0.01
+                  sum without (%(etcd_instance_labels)s) (rate(etcd_network_peer_sent_failures_total{%(etcd_selector)s}[%(scrape_interval)ss])) > 0.01
```

Comment:
Do you need a `*4` in here?

Reply:
It calculated in https://github.com/etcd-io/etcd/pull/12177/files#diff-8d107aa403e491ffbc966fb052bba76dR31 so no need to multiply it again.

Reply:
Hmm.  Can we give the multiplied form a different name like `range_interval`?

Reply:
The arithmetic is done down below because I couldn't figure out how to do it inline within the `|||` text block interpolation — see L31

```
scrape_interval: $._config.scrape_interval_seconds*4}
```

Reply:
`range_interval` sounds good to me

Reply:
I changed it to `network_failure_range` but can change it to `range_interval` if you think that's better